src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @NotNull
+    @Min(0)
     private int capacity;
```

Comment:
Change the type of 'capacity' from 'int' to 'Integer'
1. It's better to have objects fields in DTO in order to easily differentiate not provided data (null). Primitive will have default value in such case
2. '@NotNull' annotation cannot be applied to a primitive data type like 'int'.